models/productDao.js

```diff
@@ -0,0 +1,39 @@
+const appDataSource = require("./appDataSource");
+
+const getProducts = async (req) => {
```

Comment:
```suggestion
const getProducts = async (req) => {
```
마찬가지입니다. req를 그대로 받는것이 아닌 controller에서 명확한 변수명을 선언하여 이를 받도록 해주세요.

또한 지금 함수에서 req라는 이름으로 받은 argument를 내부 로직 중 어디에도 쓰고 있지 않습니다. 다시 확인해주세요

Reply:
알겟습니다 !